std/examples/xeval.ts

```diff
@@ -0,0 +1,165 @@
+import { parse } from "../flags/mod.ts";
```

Comment:
Can you add a comment with a little example of using xeval?

Use https://deno.land/std/examples/xeval.ts in the example so its easy to copy and paste for people

Reply:
Using that url would fail before `examples/xeval.ts` itself is merged

Reply:
Yes just a command line example and a description of what this does. Maybe best if the example acts on a well known file like /etc/passwd so it will work out of the box